bitcoin/src/blockdata/script/mod.rs

```diff
@@ -115,6 +115,8 @@ pub fn write_scriptint(out: &mut [u8; 8], n: i64) -> usize {
 /// don't fit in 64 bits (for efficiency on modern processors) so we
 /// simply say, anything in excess of 32 bits is no longer a number.
 /// This is basically a ranged type implementation.
+///
+/// This code is based on the `CScriptNum` constructor in Bitcoin Core (see `script.h`).
```

Comment:
That's an implementation detail. I wouldn't use doc comment.

Reply:
Agreed, but I'm not going to block the PR on it.